src/ext/graphql/transmodelapi/schema.graphql

```diff
@@ -1161,6 +1194,16 @@ type ValidityPeriod {
   startTime: DateTime
 }
 
+"Description of a travel between three or more places."
+type ViaTrip {
+  "A list of routing errors, and fields which caused them"
+  routingErrors: [RoutingError!]!
+  "A list of lists of which indices of the next segment the trip pattern can be combined with"
+  tripPatternCombinations: [[[Int!]!]!]!
+  "A list of lists of the trip patterns for each segment of the journey"
+  tripPatterns: [[TripPattern!]!]!
```

Comment:
We should have Collection types here, so it become clear what things are. This will also make it possible add information to the relations.

Something like this, maybe:
```
type ViaTrip {
  // One element per via location
  tripPatternCombinations : [ViaTripPatternCombinations!]!

  // One element per leg/segment
  tripPatternsPerLeg : [ViaTripPatternLegs!]!
}


type ViaTripPatternCombinations {
  pairs : [ViaTripPatternPair!]!
}

type ViaTripPatternPair{
  from : int
  to : int
}

type ViaTripPatternLegs {
  tripPatterns : [TripPattern!]!
}
```

Reply:
We should standardize on using either `Leg` or `Segment`, both works for me.

Reply:
Done, went with `Segment`